docs/core/getting-started/install/opensuse.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Install for openSUSE
+description: Install for openSUSE
+keywords: .NET, .NET Core, C#, Getting Started, Acquisition, Cross Platform, Linux, openSUSE
+author: kendrahavens
+ms.author: kendrahavens
+ms.date: 12/19/2016
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: 
+---
+
+# Install for openSUSE
+## 1. Install .NET Core SDK
+- Before you start, please remove any previous versions of .NET Core from your system.
+- In order to install .NET Core 1.1 on openSUSE, first you need to get the prerequisites and then you download the .NET Core SDK binaries, extract them onto your system and put dotnet onto your PATH.
+- .NET Core 1.1 is the latest version. For long term support versions and additional downloads check the [all Linux downloads](https://www.microsoft.com/net/download/linux) section.
+
+openSUSE 13.2
```

Comment:
Yeah, putting them in a list makes them blend in too much with the bulleted list they follow. I'll try 3 hashtags.